aspnet/web-api/overview/security/external-authentication-services/samples/sample1.cs

```diff
@@ -1,55 +1,74 @@
 using System;
```

Comment:
It looks like there are substantial changes in this file -- where did all the new code come from?